usr/local/bin/easyrule

```diff
@@ -91,11 +91,24 @@ function is_specialnet($net) {
 
 
 if (($argc > 1) && !empty($argv[1])) {
+
+	# Automagically derive an alternate alias name from the scripts name
+	# This allows for using alternate alias lists with just a symlink
```

Comment:
Sorry about the long delay. Please just fix this comment style using /* */, that is recommended way for multiline comments and I'll get it merged